### ISSUE TITLE
ClassicAra does not load using the current model.

### ISSUE THREAD

**Issue by aochoam** [QueensGambit/CrazyAra#155]
**Executable:** ClassicAra.exe 0.9.5

**Error:** ClassicAra does not load using the current model.

**Used Models**
C:\Arena\Engines\ClassicAra\model\ClassicAra\chess>dir
 El volumen de la unidad C no tiene etiqueta.
 El número de serie del volumen es: 2A49-5FC0

 Directorio de C:\Arena\Engines\ClassicAra\model\ClassicAra\chess

01/09/2021  10:26 p. m.    <DIR>          .
01/09/2021  10:26 p. m.    <DIR>          ..
**01/09/2021  10:26 p. m.        17.170.482 model-1.19414-0.585-0513-v3.0-bsize-1.onnx
01/09/2021  10:26 p. m.        17.170.482 model-1.19414-0.585-0513-v3.0-bsize-16.onnx
01/09/2021  10:26 p. m.        17.170.482 model-1.19414-0.585-0513-v3.0-bsize-8.onnx
01/09/2021  10:26 p. m.        17.136.741 model-1.19414-0.585-v3.0-0513.params
01/09/2021  10:26 p. m.            83.627 model-1.19414-0.585-v3.0-symbol.json**
01/09/2021  10:26 p. m.               743 model_7_wdlp_head_7_epochs.txt

**Execution:**
:\Arena\Engines\ClassicAra>ClassicAra.exe

                                  _
                   _..           /   ._   _.  _        /\   ._   _.
                 .' _ `\         \_  |   (_|  /_  \/  /--\  |   (_|
                /  /e)-,\                         /
               /  |  ,_ |                    __    __    __    __
              /   '-(-.)/          bw     8 /__////__////__////__////
            .'--.   \  `                 7 ////__////__////__////__/
           /    `\   |                  6 /__////__////__////__////
         /`       |  / /`\.-.          5 ////__////__////__////__/
       .'        ;  /  \_/__/         4 /__////__////__////__////
     .'`-'_     /_.'))).-` \         3 ////__////__////__////__/
    / -'_.'---;`'-))).-'`\_/        2 /__////__////__////__////
   (__.'/   /` .'`                 1 ////__////__////__////__/
    (_.'/ /` /`                       a  b  c  d  e  f  g  h
      _|.' /`
jgs.-` __.'|  Developers: Johannes Czech, Moritz Willig, Alena Beyer
    .-'||  |  Source-Code: QueensGambit/CrazyAra (GPLv3-License)
       \_`/   Inspiration: A0-paper by Silver, Hubert, Schrittwieser et al.
              ASCII-Art: Joan G. Stark, Chappell, Burton

uci
id name ClassicAra 0.9.5 (Aug 23 2021)
id author Johannes Czech and CrazyAra developers (see AUTHORS file)
option name Allow_Early_Stopping type check default true
option name Batch_Size type spin default 16 min 1 max 8192
option name Centi_CPuct_Init type spin default 250 min 1 max 99999
option name Centi_Dirichlet_Epsilon type spin default 0 min 0 max 99999
option name Centi_Dirichlet_Alpha type spin default 20 min 1 max 99999
option name Centi_Epsilon_Checks type spin default 1 min 0 max 100
option name Centi_Epsilon_Greedy type spin default 5 min 0 max 100
option name Centi_Node_Temperature type spin default 170 min 1 max 99999
option name Centi_Q_Value_Weight type spin default 100 min 0 max 99999
option name Centi_Q_Veto_Delta type spin default 40 min 0 max 99999
option name Centi_Quantile_Clipping type spin default 25 min 0 max 100
option name Centi_Random_Move_Factor type spin default 0 min 0 max 99
option name Centi_Temperature type spin default 170 min 0 max 99999
option name Centi_Temperature_Decay type spin default 92 min 0 max 100
option name Centi_U_Init_Divisor type spin default 100 min 1 max 99999
option name Centi_Virtual_Loss type spin default 100 min 0 max 99999
option name Context type string default cpu
option name CPuct_Base type spin default 19652 min 1 max 99999
option name First_Device_ID type spin default 0 min 0 max 99999
option name Fixed_Movetime type spin default 0 min 0 max 99999999
option name Last_Device_ID type spin default 0 min 0 max 99999
option name Log_File type string default
option name MCTS_Solver type check default true
option name Model_Directory type string default model/ClassicAra/chess
option name Move_Overhead type spin default 20 min 0 max 5000
option name MultiPV type spin default 1 min 1 max 99999
option name Nodes type spin default 0 min 0 max 99999999
option name Nodes_Limit type spin default 0 min 0 max 999999999
option name Precision type combo default float32 var float32 var int8
option name Reuse_Tree type check default true
option name Temperature_Moves type spin default 0 min 0 max 99999
option name Use_NPS_Time_Manager type check default true
option name Search_Type type combo default mcgs var mcgs var mcts
option name Simulations type spin default 0 min 0 max 99999999
option name SyzygyPath type string default <empty>
option name Threads type spin default 2 min 1 max 512
option name Timeout_MS type spin default 0 min 0 max 99999999
option name UCI_Variant type combo default chess var chess var chess
option name Use_Raw_Network type check default false
uciok
isready
info string json file: model/ClassicAra/chess/model-1.19414-0.585-v3.0-symbol.json
info string Loading the model from model/ClassicAra/chess/model-1.19414-0.585-v3.0-symbol.json
[08:04:19] C:\lib\incubator-mxnet\src\nnvm\legacy_json_util.cc:204: **Warning: loading symbol saved by MXNet version 10501 with lower version of MXNet v10500. May cause undefined behavior. Please update MXNet if you encounter any issue
info string Loading the model parameters from: model/ClassicAra/chess/model-1.19414-0.585-v3.0-0513.params**

C:\Arena\Engines\ClassicAra>pause

**Comment by QueensGambit** [comment]
Hello @aochoam ,
yes, it looks like the MXNet library provided in the Windows version is unable to load the new **ClassicAra-sl-model-wdlp-rise3.3-input3.0** model.
I am sorry for the inconvenience. All other models (_MultiAra_ models, and _CrazyAra_ model) should load fine in the Windows CPU version.
However, the Mac and Linux CPU version should be able to load the new _ClassicAra_ model.
I tried to build a newer MXNet backend library on Windows but it failed due to several out of memory compiler errors for me, despite having 32 GiB RAM.

As an alternative, I started to add a OpenVino backend.
The OpenVino API can be used with CPUs (AMD & Intel), the Intel Neural Compute Stick and Intel GPUs.
* https://github.com/QueensGambit/CrazyAra/pull/153

So there will probably be a **release 0.9.5.post0** soon to provide a working _ClassicAra_ CPU Windows version.
